README.md

```diff
@@ -11,6 +11,7 @@ Build and test status of specific cards:
 | Cards                                                               | Status                                                                                                                            |
 |---------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
 | CAC                                                                 | [![CAC](https://gitlab.com/redhat-crypto/OpenSC/badges/cac/build.svg)](https://gitlab.com/redhat-crypto/OpenSC/pipelines)         |
+| [virt_CACard](https://github.com/PL4typus/OpenSC/tree/virt_cacard)                                                         | [![virt_cacard](https://gitlab.com/PL4typus/OpenSC/badges/virt_cacard/pipeline.svg)](https://gitlab.com/PL4typus/OpenSC/pipelines)        |
```

Comment:
This should probably link now to the travis results.

Reply:
The travis badge is for the whole build, and not a specific job. Since there is already a travis badge for OpenSC, should I leave the one pointing specifically to virtual CAC testing, or remove it altogether ?

Reply:
I agree that the image should point to the Travis result. Hopefully, the service will allow linking to a specific job sometime soon, which would the link make more specific then.